app/src/app_ios.mm

```diff
@@ -277,7 +281,7 @@ static void PlatformOptionsToAppOptions(FIROptions* platform_options, AppOptions
 
 App* App::GetInstance(const char* name) { return app_common::FindAppByName(name); }
 
-void App::RegisterLibrary(const char* library, const char* version) {
+void App::RegisterLibrary(const char* library, const char* version, void* platform_resource) {
```

Comment:
If it's not used on this platform, do the /* platform_resource */ thing.

Reply:
Done. Also changed for desktop and stub